frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/index.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { useEffect } from "react";
 import { Box } from "@chakra-ui/react";
 import PaymentSummary from "./PaymentSummary";
-import ReviewInformation from "./ReviewInformation";
```

Comment:
How did you make it compile even though the component isn't used?

Reply:
review registration is currently commented out. We add this in tejas's PR